qubesappmenus/__init__.py

```diff
@@ -42,6 +42,11 @@
 
 basedir = os.path.join(xdg.BaseDirectory.xdg_data_home, 'qubes-appmenus')
 
+def vm_name_escape(vm_name: str) -> str:
+    """Escape a VM name for use in a .desktop file name"""
+    return ('_' + vm_name.replace('_', '_u')
```

Comment:
why starting with `_`? is it trying to prefix with delimited in this function itself? then it doesn't look to be working as expected, as you get double delimiters in some places (like `org.qubes-os.vm._vmname`)

Reply:
If I recall correctly, it is to avoid collisions with old-syntax desktop files.  VM names cannot start with `_`, so old-syntax desktop files never had `_` in that position.  The purpose of the escaping is entirely to prevent name collisions.